crates/red_knot_python_semantic/resources/mdtest/sys_version_info.md

```diff
@@ -0,0 +1,131 @@
+# `sys.version_info`
+
+## The type of `sys.version_info`
+
+The type of `sys.version_info` is `sys._version_info`. This is quite a complicated type in typeshed,
+so there are many things we don't fully understand about the type yet:
```

Comment:
… but would you eventually expect a different output in the snippet below? As in: is this a TODO comment?

Reply:
> … but would you eventually expect a different output in the `reveal_type` below?

No. But it _is_ the reason why there are several other `Type::Todo`s and TODO comments in the file!

It's also the reason why I haven't implemented _more_ special-casing, if that makes sense. What I mean is that we _could_ add special-casing for this class in several other places (`Type::iterate()`, the attribute access for attributes other than `major` and `minor`, etc.). But doing that would be a bit silly IMO, because we'll naturally add support for these without any special-casing for the `sys._version_info` class when we add support for more typing features in the future.

Anyway, I'll clarify this sentence, thanks!